controllers/operatorconfigmap_controller.go

```diff
@@ -131,8 +131,17 @@ func (c *OperatorConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		),
 	)
 
+	servicePredicate := builder.WithPredicates(
+		predicate.NewPredicateFuncs(
+			func(obj client.Object) bool {
+				return obj.GetNamespace() == c.OperatorNamespace
+			},
+		),
+	)
```

Comment:
Why do we need this predicate if you have an ownership watch?

Reply:
we get reconcile events based on Kind and this controller is for a configmap and w/o this predicate we'll get an event for all services which was owned by a configmap.